src/containers/RepositoriesContainer/RepositoriesContainer.js

```diff
@@ -0,0 +1,45 @@
+import React, { PropTypes, Component } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+// Actions
+import * as actionCreators from './RepositoriesDucks.js';
+
+// Components
+import RepoList from '../../components/RepoList/';
+
+const mapStateToProps = (state) => ({
+  isLoading: state.repositories.isLoading,
+  repos: state.repositories.list
+});
+
+const mapDispatchToProps = (dispatch) => bindActionCreators(actionCreators, dispatch);
+
+class RepositoriesContainer extends Component {
+
+  static propTypes = {
+    isLoading: PropTypes.bool.isRequired,
+    repos: PropTypes.array,
+    load: PropTypes.func.isRequired
+  };
+
+  constructor (props) {
+    super(props);
+
+    if (typeof props.repos === 'undefined') {
+      props.load();
+    }
+  }
+
+  render () {
+    const { isLoading, repos } = this.props;
+
+    if (repos && repos.length > 0) {
+      return <RepoList repos={repos} />;
+    }
+
+    return isLoading ? <p>Loading...</p> : <p>No repositories to display.</p>;
```

Comment:
loading in the containers can be a generic util. Need to think how to approach it best.
But `No repositories to display.` should definitely belong to RepoList imho. Container doesn't care.